docs/operator-manual/upgrading/2.3-2.4.md

```diff
@@ -69,20 +69,77 @@ server's key signature algorithms configuration.
 > authentication via the HostkeyAlgorithms and PubkeyAcceptedAlgorithms
 > options. For example, the following stanza in ~/.ssh/config will enable
 > RSA/SHA1 for host and user authentication for a single destination host:
-> 
-> ```
-> Host old-host
->     HostkeyAlgorithms +ssh-rsa
->     PubkeyAcceptedAlgorithms +ssh-rsa
-> ```
-> 
+ 
+ ```
+ Host old-host
+     HostkeyAlgorithms +ssh-rsa
+     PubkeyAcceptedAlgorithms +ssh-rsa
+ ```
+ 
 > We recommend enabling RSA/SHA1 only as a stopgap measure until legacy
 > implementations can be upgraded or reconfigured with another key type
 > (such as ECDSA or Ed25519).
 
 To apply this to Argo CD, you could create a ConfigMap with the desired ssh config file and then mount it at 
 `/home/argocd/.ssh/config`.
 
+### argocd-ssh-cm.yaml
```

Comment:
Can we nest these two sections under a header called "Example workaround with Kustomize (change details to match your environment)" or something like that?